src/browser/pages/login.html

```diff
@@ -25,7 +25,7 @@
       <div class="card-box">
         <div class="header">
           <h1 class="main">{{WELCOME_TEXT}}</h1>
-          <div class="sub">Please log in below. {{PASSWORD_MSG}}</div>
```

Comment:
Part of me thinks we should keep the " " here between `{{I18N_LOGIN_BELOW}}{{PASSWORD_MSG}}` since I could see someone forgetting to add it to the `locales` file.